training/src/main/java/wolox/training/models/Book.java

```diff
@@ -12,40 +14,51 @@
 import wolox.training.exceptions.BookNotOwnedByUserException;
 
 @Entity
+@ApiModel(description = "Books from the TrainingApi")
 public class Book {
 
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private long id;
 
     @Column
+    @ApiModelProperty(notes = "The book genre: horror, comedy, drama, etc.")
     private String genre;
 
     @Column(nullable = false)
+    @ApiModelProperty(notes = "The book author, e.g., Borges.")
     private String author;
 
     @Column(nullable = false)
+    @ApiModelProperty(notes = "The images of the book.")
     private String image;
 
     @Column(nullable = false, unique = true)
+    @ApiModelProperty(notes = "The title of the book, e.g., The Wolox Chronicles.")
     private String title;
 
     @Column(nullable = false)
+    @ApiModelProperty(notes = "The subtitle of the book, e.g., The tale of an old pull request.")
     private String subtitle;
 
     @Column(nullable = false)
+    @ApiModelProperty(notes = "The publisher of the book, e.g., Salamandra.")
     private String publisher;
 
     @Column(nullable = false)
+    @ApiModelProperty(notes = "The year of publication: 2019, 2018, etc.")
     private String year;
 
     @Column(nullable = false)
+    @ApiModelProperty(notes = "The number of pages the book has.")
     private int pages;
 
     @Column(nullable = false)
+    @ApiModelProperty(notes = "The number of pages the book has.")
     private String isbn;
```

Comment:
The api model property for isbn is not right, the isbn is the property used to identify any book (Like a unique document number, but for books)

Reply:
yes, sorry it was a copy mistake. My bad :(

Reply:
Fixed

Reply:
Great! No problem :)